### ISSUE TITLE
Timeline view has incorrect scroll position when there are many events with multiline titles

### ISSUE THREAD

**Issue by wacki4** [fullcalendar/fullcalendar#4443]
https://codepen.io/anon/pen/xmzzXz?&editable=true&editors=001

When calendar initalize - You can see that resources are correct on the top. But, after loading many events - vertical scroll moves down, which made us always have to scroll it to top.

Moving next/prev works correctly when we scroll to resource which is more at bottom, but on top scroll always make a problem and moves down.

Data in link works for example for 8-15 January to reproduce a problem.

**Comment by acerix** [comment]
Thanks for the details report.

I can see that by pressing next/prev the calendar scrolls down when it should stay scrolled to the top, is that the issue you mean?

I think the issue is the line breaks in your event titles since the timeline view assumes each event is only 1 line.

**Comment by wacki4** [comment]
Yes, exacly it is a problem. See that if you scroll down, it correctly remeber position. It looks like scheduler treat it like one line of event, not accept many events.

**Comment by szuba1** [comment]
I have the same issue - is there a chance to solve it in the near future?

**Comment by arshaw** [comment]
I'm having trouble reproducing this. Whether I'm initially scrolled down or not, the scroll is correctly maintained across prev/next navigations. Could someone please post a video or a screen GIF of the problem happening?

**Comment by wacki4** [comment]
https://youtu.be/3UjM4_62pW8 here some video of problem.

**Comment by arshaw** [comment]
thanks @wacki4 ,

also, after modifying your original codepen to have the defaultDate be in view, i can see the problem: https://codepen.io/arshaw/pen/mNqYGQ

we still need to recreate this in v4 though. can someone modify the codepen to use v4 plz?

**Comment by wacki4** [comment]
Here You have one:

https://codepen.io/wacki4/pen/bGbodja

**Comment by acerix** [comment]
It looks like whatever resource that is at the top now stays at the top after changing the day, how are you expecting it to work differently? If you see something different, what are the steps to reproduce the issue?

It would help to hardcode the events in your demo instead of loading from your server so its more consistent.

**Comment by wacki4** [comment]
@acerix It is not an error i'm reffering to. When we have many events on same day/hour they are shown in rows - when You switch between two views - scroll is messing around and when having it on top on one view, on second one - it shows only last row of these events from first resource - everything else is outside viewport. You can see it on video provided above.

**Comment by wacki4** [comment]
Even on start of last codepen link - You can see that scroll is not on the top, it shows only last event from first resource.

**Comment by acerix** [comment]
Sorry, I was mistaken with another issue.  I can see the issue after following those steps.

**Comment by arshaw** [comment]
i now understand the bug (while looking at [this repro](https://codepen.io/wacki4/pen/bGbodja?editors=0010))... things are initially scrolled down even when the user has not explicitly scrolled down. this is mainly due to the fact that the events dynamically load

**Comment by wacki4** [comment]
How is going repairing this error?

**Comment by VShubham100** [comment]
any solution to this?

**Comment by leemorris** [comment]
Is there an update?

**Comment by sehaj23** [comment]
Any solution for this?

**Comment by arshaw** [comment]
A fix has been implemented in [v7.0.0-beta.4](https://github.com/fullcalendar/fullcalendar-workspace/releases/tag/v7.0.0-beta.4)

Updated repro:
https://codepen.io/arshaw/pen/qEWxXPp?editors=0010

Could someone please verify?
